src/index.js

```diff
@@ -30,6 +32,9 @@ const config = getConfig();
 // Convenience wrapper around Prettier, so that config doesn't have to be
 // passed every time.
 const prettify = buildPrettifier(config.prettierConfig);
+const prettifyCss = buildPrettifier(Object.assign({}, config.prettierConfig, {
+  parser: 'postcss'
+}));
```

Comment:
Awesome! Thanks so much for prettifying the CSS, had no idea Prettier could do that 🔥 🔥 🔥